examples/hf_serialization.py

```diff
@@ -242,37 +325,36 @@ def hf_main(args):
     logger.info("GPU: " + utils.get_gpu_name())
     logger.info("PYTHON USED RAM: " + utils.get_mem_usage())
 
-    serialize_model(model, model_config, output_prefix)
-
-    tokenizer = AutoTokenizer.from_pretrained(args.input_directory)
-    tokenizer.save_pretrained(output_prefix)
+    serialize_model(model, model_config, output_prefix, None, args.force)
 
     if args.validate:
-        del model, model_config
+        # Not sure if this part is needed as, although I doubt it,
+        # I'm not absolutely certain whether or not TensorSerializer
+        # changes the model's state in any way
```

Comment:
`TensorSerializer` does not change the model's state (unless you pass a special flag to delete tensors after writing them, or an error occurs in the middle of model encryption). It used to by default a long time ago, which might be why that is there.

Reply:
Okay, I won't reinstantiate `model` in that case. Will resolve when this is fixed.

Reply:
This is resolved in #76, whenever that is merged